app.js

```diff
@@ -50,7 +50,7 @@ app.all("*", function(req, res) {
 
 app.use((err, req, res) => {
 	if(err.name == "UnauthorizedError"){
-		return apiResponse.unauthorizedResponse(res, err.message);
+		apiResponse.unauthorizedResponse(res, err.message);
```

Comment:
A return will again pass the error to next middleware, if nothing handles then it will be thrown as uncaught.